src/domain/bulkTransfer/transformer.js

```diff
@@ -0,0 +1,142 @@
+/*****
```

Comment:
I am thinking we should move this to a common lib.

Aka, anything that you find common between the bulk-api-adapter and the ml-api-adapter should probably be moved or at least considered to be moved into a common lib.

Let me know what you think about this?

At the very least, we should create a story to do this later if we are unable to do it right now.

Reply:
I agree - let's cover it as a task in https://github.com/mojaloop/project/issues/848